Pre-course 1 exercise completed: Ready for review

gunjan004 opened the pull request.

Lohith-Aswathappa reviewed (commented):
Good work, Gunjan!
include time and space complexity and also comment your code

Lohith-Aswathappa commented:
Good work, Gunjan!
include time and space complexity and also comment your code

gunjan004 commented:
> Good work, Gunjan!
> include time and space complexity and also comment your code

Hello lohith, Sure will do so. Also, comment the code as in what can you please elaborate? Do I need to comment the solution and create pull request again?

gunjan004 commented:
> Good work, Gunjan!
> include time and space complexity and also comment your code

Hi Lohith, I added complexities, I already added comments on lines to explain and I have added few explanations of the code as well, was there anything else that you meant by comment the code? Please let me know if I have missed anything.
